pullrequest.rst

```diff
@@ -221,6 +221,9 @@ commit, a new paragraph(s) can be added to explain in proper depth what has
 happened (detail should be good enough that a core developer reading the
 commit message understands the justification for the change).
 
+Check :ref:`the git bootcamp<accepting-and-merging-a-pr>` for further
```

Comment:
You should add a space after bootcamp (not sure if it's required, but for consistency it should be added regardless).